website/content/en/docs/building-operators/helm/migration.md

```diff
@@ -1,12 +1,10 @@
 ---
 title: Migrating Helm-based Legacy Projects
 linkTitle: Migrating Legacy Projects
-weight: 300
-description: Instructions for migrating a legacy Helm-based project to use the new Kubebuilder-style layout.
+weight: 3
```

Comment:
I intentionally made these weights separated by 100 so that we can add new pages anywhere in the list without changing the others' weights.

I think revert this back to 300.